server/data/apps.json

```
@@ -587,6 +587,10 @@
         "elementName": "Value",
         "elementURL": "https://curiocaster.com/"
       },
+      {
+        "elementName": "Live",
+        "elementURL": "https://twitter.com/Podverse/status/1521198937742487553"
```

Shouldn't this be curiocaster.com? Looks like you had it linked to a podverse tweet.
```suggestion
        "elementURL": "https://curiocaster.com/"
```